packages/menu-surface/package.json

```diff
@@ -19,7 +19,8 @@
   "dependencies": {
     "@material/menu-surface": "^0.41.0",
     "classnames": "^2.2.5",
-    "react": "^16.4.2"
+    "react": "^16.4.2",
+    "react-dom": "16.4.2"
```

Comment:
@ben-mckernan can you also add the `^` to react-dom please?

Reply:
Yes, of course. Good catch!